crates/cxx-qt-lib/src/types/qdate.cpp

```diff
@@ -7,6 +7,21 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
 #include "cxx-qt-lib/include/qdate.h"
 
+#include "assertion_utils.h"
+
+#include <cstdint>
+
+// QDate has a single 64-Bit integer member
+// https://code.qt.io/cgit/qt/qtbase.git/tree/src/corelib/time/qdatetime.h?h=v5.15.6-lts-lgpl#n176
+//
+// https://code.qt.io/cgit/qt/qtbase.git/tree/src/corelib/time/qdatetime.h?h=v6.2.4#n147
+assert_alignment_and_size(QDate, alignof(std::int64_t), sizeof(std::int64_t));
+
+static_assert(std::is_trivially_copy_assignable<QDate>::value);
+static_assert(std::is_trivially_copy_constructible<QDate>::value);
+
+static_assert(std::is_trivially_destructible<QDate>::value);
```

Comment:
should we be checking for is_trivially_destructible in other trivial types? eg QPoint etc ?

Reply:
is_trivially_copyable already asserts this.
See: https://en.cppreference.com/w/cpp/language/classes#Trivially_copyable_class

Reply:
Aha

Reply:
So then can QDate just be `is_trivially_copyable` ?